src/HealthChecks.RavenDB/RavenDBHealthCheck.cs

```diff
@@ -11,20 +11,33 @@ namespace HealthChecks.RavenDB
     public class RavenDBHealthCheck : IHealthCheck
     {
         private readonly string _connectionString;
+        private readonly string[] _urls;
         private readonly string _specifiedDatabase;
 
         public RavenDBHealthCheck(string connectionString, string databaseName = default)
         {
             _connectionString = connectionString ?? throw new ArgumentNullException(nameof(connectionString));
             _specifiedDatabase = databaseName;
         }
+
```

Comment:
Hi, probably you can use only one constructor for both cases!

Reply:
Correct. I can fix this.

Reply:
Done.